packages/fxa-content-server/app/scripts/views/mixins/service-mixin.js

```diff
@@ -32,7 +33,11 @@ export default {
       service,
       serviceName: subscriptionProductName || serviceName,
       serviceLogo,
-      isSync: this.relier.isSync(),
+      // relier.isSync() doesn't appear to get set from deeplink in mobile FF
+      // so also check serviceName, see #13425
```

Comment:
> relier.isSync() doesn't appear to get set

I can't quite parse this.  Is `isSync` not there?

Reply:
This was my poor attempt at trying to summarize what I said in my longer comment earlier. No longer applicable now so it was removed.